tests/KdybyTests/CsobPaymentGateway/Payment.phpt

```diff
@@ -0,0 +1,233 @@
+<?php
+
+/**
+ * Payment
+ *
+ * @testCase
+ */
+
+namespace KdybyTests\CsobPaymentGateway;
+
+use Kdyby;
+use Kdyby\CsobPaymentGateway\Payment;
+use Tester;
+use Tester\Assert;
+
+
+
+require_once __DIR__ . '/../bootstrap.php';
+
+
+
+/**
+ * @author Jiří Pudil <me@jiripudil.cz>
+ */
+class PaymentTest extends Tester\TestCase
+{
+
+	public function testPayment()
+	{
+		$payment = new Payment('A1029DTmM7', 15000001);
+		$payment->setDttm(new \DateTime('2015-11-11 12:00:00'));
+		$payment->setDescription('Test payment');
+		$payment->setReturnUrl('https://example.com/process-payment-response');
+
+		$payment->addCartItem('Test item 1', 4200, 1);
```

Comment:
Zrovna jsem se nachytal, když jsem tenhle test použil jako základ pro implementaci. Bylo by možná lepší uvádět ceny jako `42 * 100` - pak je z toho jasně vidět, že to je v centech.